src/Tizen.NUI.WindowSystem/src/public/EdgeDragData.cs

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright(c) 2023 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using static Tizen.NUI.WindowSystem.Interop.InputGesture;
+
+namespace Tizen.NUI.WindowSystem
+{
+    /// <summary>
+    /// Represents a wrapper class for a unmanaged gesture data handle.
+    /// </summary>
+    public class EdgeDragData : IDisposable
```

Comment:
could you add the following attribute for hidden API?
`/// This class is need to be hidden as inhouse API.
[EditorBrowsable(EditorBrowsableState.Never)]`